diff --git a/app/utils/generic.py b/app/utils/generic.py
--- a/app/utils/generic.py
+++ b/app/utils/generic.py
@@ -399,6 +399,9 @@
         f"Launching the game with subprocess.Popen(): `{executable_path}` "
         f"with env_vars: {list(env_vars.keys())}, wrappers: {wrapper_commands}, args: {game_args}"
     )
+    if sys.platform == "linux":
+        game_args = [executable_path] + game_args
+        executable_path = "@steam-run@/bin/steam-run"
     pid, popen_args = launch_process(
         executable_path,
         game_args,
diff --git a/app/utils/steam/steamcmd/wrapper.py b/app/utils/steam/steamcmd/wrapper.py
--- a/app/utils/steam/steamcmd/wrapper.py
+++ b/app/utils/steam/steamcmd/wrapper.py
@@ -350,8 +350,8 @@
         script_path = self._build_download_script(first_batch)
         runner.message(f"Compiled & using script: {script_path}")
         runner.execute(
-            self.steamcmd,
-            [f'+runscript "{script_path}"'],
+            "@steam-run@/bin/steam-run",
+            [self.steamcmd, f'+runscript "{script_path}"'],
             total,
         )
